zerver/webhooks/groove/doc.md

```diff
@@ -0,0 +1,39 @@
+Zulip supports integration with Groove and can notify you on any events
+happening on your Groove dashboard.
+
+1. {!create-stream.md!}
+
+2. {!create-bot-construct-url-indented.md!}
+
+3. Next, on your Groove dashboard, go to **Settings** tab
+
+    ![](/static/images/integrations/groove/001.png)
+
+4. From there, go to the **API** menu under **Company**.
+
+    ![](/static/images/integrations/groove/002.png)
+
+5. On the **Webhooks** option, open the **Add Webhook**
+   dropdown and select the event that you wish to be notified.
```

Comment:
Replace `that you wish to be notified` with `that you wish to be notified of`. :)